build.gradle

```diff
@@ -51,30 +54,30 @@ jar {
 }
 
 dependencies {
-    compile 'org.antlr:antlr4-runtime:4.7.2'
-    compile 'org.slf4j:slf4j-api:' + slf4jVersion
-    compile 'com.graphql-java:java-dataloader:2.2.3'
-    compile 'org.reactivestreams:reactive-streams:' + reactiveStreamsVersion
+    implementation 'org.antlr:antlr4-runtime:4.8'
```

Comment:
@bbakerman this is now explicitly API vs implementation detail. I am not completely sure this is correct.